index.js

```diff
@@ -8,6 +8,7 @@ var defaults = {
   trustAzureHeader: false,
   port: 443,
   hostname: null,
+  skipPort: false,
```

Comment:
I'm OK with it because:

- I don't believe it will break anyones application (thanks to this spec https://tools.ietf.org/html/rfc2818#section-2.3)
- Since this will provide new API I'll tag next release as `1.1.0` anyway